MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -800,12 +830,17 @@ - (void)removeAnalyzerFile {
       MSLogError([MSCrashes logTag], @"Couldn't remove analyzer file at %@ with error %@.", self.analyzerInProgressFile,
                  error.localizedDescription);
     }
+  } else if (error) {
+    MSLogWarning([MSCrashes logTag], @"Check \"%@\" error: %@", self.analyzerInProgressFile, error.localizedDescription);
```

Comment:
I really doubt that we need this log, @clpolet @TroubleMakerBen. If yes, revise the message.